tests/OmniSharp.Roslyn.CSharp.Tests/SignatureHelpFacts.cs

```diff
@@ -115,7 +115,7 @@ public SignatureHelpFacts(ITestOutputHelper output)
     public static void Main(){
         Foo($$);
     }
-    pubic static Foo(bool b, int n = 1234)
+    public static Foo(bool b, int n = 1234)
```

Comment:
have we verified that this code was not intentionally invalid previously?

Reply:
No, didn't verify that. Should I revert it back to the previous state ?

Reply:
I wouldn't think so. This doesn't look intentional to me based on the test name.

Reply:
You should look at the test and decide whether the typo was intentional :)

Reply:
My guess, based on what's being tested, is that it was unintentional.